File: javascript/Q3.js
var s1=10;
var s2=15;
var s3=12;

function trai(){
    if (s1==s2 && s2==s3){
        console.log("Equilteral")
    }else if (s1==s2 || s2==s3){
        console.log("Isosceles")
    }else{
        console.log("Scalene")
    }
}

trai()

File: C - language/Q3.c
#include <stdio.h>

int main() {
    int s1=10;
    int s2=20;
    int s3=10;
    
    if (s1==s2 && s2==s3){
        printf("Equilteral");
    }else if (s1==s2 || s2==s3 || s1==s3){
        printf("Isosceles");
    }else{
        printf("Scalene");
    }

    return 0;
}
